k210.svd

```diff
@@ -1011,11 +1011,110 @@
             <groupName>GPIO</groupName>
             <baseAddress>0x50200000</baseAddress>
             <registers>
-                <!-- TODO -->
                 <register>
-                    <name>dummy</name>
-                    <description>Dummy register: this peripheral is not implemented yet</description>
+                    <name>data_output</name>
+                    <description>Data (output) registers</description>
                     <addressOffset>0x00</addressOffset>
+                    <fields>
+                        <field><name>pin0</name><lsb>0</lsb><msb>0</msb></field>
+                        <field><name>pin1</name><lsb>1</lsb><msb>1</msb></field>
+                        <field><name>pin2</name><lsb>2</lsb><msb>2</msb></field>
+                        <field><name>pin3</name><lsb>3</lsb><msb>3</msb></field>
+                        <field><name>pin4</name><lsb>4</lsb><msb>4</msb></field>
+                        <field><name>pin5</name><lsb>5</lsb><msb>5</msb></field>
+                        <field><name>pin6</name><lsb>6</lsb><msb>6</msb></field>
+                        <field><name>pin7</name><lsb>7</lsb><msb>7</msb></field>
+                    </fields>
+                </register>
+                <register>
+                    <name>direction</name>
+                    <description>Data direction registers</description>
+                    <addressOffset>0x04</addressOffset>
+                    <fields>
+                        <field><name>pin0</name><lsb>0</lsb><msb>0</msb></field>
+                        <field><name>pin1</name><lsb>1</lsb><msb>1</msb></field>
```

Comment:
Could you add enumerations for `direction`?

Reply:
done!